src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -308,29 +308,53 @@ async Task<T> IDurableOrchestrationContext.CreateTimer<T>(DateTime fireAt, T sta
         {
             this.ThrowIfInvalidAccess();
 
+            DateTime intervalFireAt = fireAt;
+            TimeSpan longRunningTimerIntervalLength = TimeSpan.FromDays(3);
+
             if (!this.durabilityProvider.ValidateDelayTime(fireAt.Subtract(this.InnerContext.CurrentUtcDateTime), out string errorMessage))
             {
-                throw new ArgumentException(errorMessage, nameof(fireAt));
+                intervalFireAt = this.InnerContext.CurrentUtcDateTime.AddDays(longRunningTimerIntervalLength.TotalDays);
```

Comment:
Instead of `.AddDays(longRunningTimerIntervalLength.TotalDays)`, just do `.Add(longRunningTimerIntervalLength)`. Same comment in the places below where you are also using `AddDays`.